### DISCUSSION TITLE
forecast missing?

### DISCUSSION THREAD

**Discussion by oldwindmill** [amberelectric/public-api#93] [General]
earlier. a spike was predicted, but now i only see now and next prices...(that is, the 12 hour forecast has disappeared...) (both web and app).

i'm hoping that i will continue to at least get now and next, and that the pricing after 4pm doesnt disappear....

**Comment by joho** [comment]
We are having some technical issues with the data feeds we use to generate price forecasts. We'll update here later when we have a diagnosis and/or fix.

**Comment by oldwindmill** [comment]
seems to be good now.

**Comment by joho** [comment]
We have forecasts back in the feed, but they're not the most up date data available.

AEMO is publishing forecasts into NEMWEB with intermittent delays, and so the prices in the app and via API are still lagging a bit. We're looking into a workaround to get the data from another source.
